==========
References
==========

Abbott, C., Coulson, M., Gagné, N., Lacoursière, A., Bajno, R., Dietrich, C., & May-McNally, S.
(2021). Guidance on the use of targeted environmental DNA (eDNA) analysis for the
management of aquatic invasive species and species at risk. Canadian Science
Advisory Secretariat (CSAS).

Ando, H., Mukai, H., Komura, T., Dewi, T., Ando, M., & Isagi, Y. (2020). Methodological
trends and perspectives of animal dietary studies by non-invasive fecal DNA
metabarcoding. Environmental DNA, 2(4), 391-406. https://doi.org/10.1002/edn3.117

Allan, E. A., Zhang, W. G., C Lavery, A., & F Govindarajan, A. (2021). Environmental
DNA shedding and decay rates from diverse animal forms and thermal regimes.
Environmental DNA, 3(2), 492-514. https://doi.org/10.1002/edn3.141

Antich, A., Palacin, C., Wangensteen, O. S., & Turon, X. (2021). To denoise or to
cluster, that is not the question: optimizing pipelines for COI metabarcoding and
metaphylogeography. BMC Bioinformatics, 22, 1-24.
https://doi.org/10.1186/s12859-021-04115-6

Baillie, S. M., McGowan, C., May-McNally, S., Leggatt, R., Sutherland, B. J., & Robinson,
S. (2019). Environmental DNA and its applications to Fisheries and Oceans Canada:
National Needs and Priorities. Fisheries and Oceans Canada= Pêches et Oceans
Canada.

Barnes, M. A., Turner, C. R., Jerde, C. L., Renshaw, M. A., Chadderton, W. L., & Lodge,
D. M. (2014). Environmental conditions influence eDNA persistence in aquatic
systems. Environmental Science & Technology, 48(3), 1819-1827.
https://doi.org/10.1021/es404734p.

Bellemain, E., Carlsen, T., Brochmann, C., Coissac, E., Taberlet, P., & Kauserud, H.
(2010). ITS as an environmental DNA barcode for fungi: An in silico approach reveals
potential PCR biases. BMC Microbiology, 10, 1-9. https://doi.org/10.1186/1471-2180-10-189

Beng, K. C., & Corlett, R. T. (2020). Applications of environmental DNA (eDNA) in
ecology and conservation: opportunities, challenges and prospects. Biodiversity and
Conservation, 29(7), 2089-2121. https://doi.org/10.1007/s10531-020-01980-0

Benson, D. A., Karsch-Mizrachi, I., Lipman, D. J., Ostell, J., & Sayers, E. W. (2009).
GenBank. Nucleic Acids Research, 37(suppl_1), D26-D31.
https://doi.org/10.1093/nar/gkm929.

Berry, O. F., Jarman, S. N., & Holleley, C. E. (Eds.). (2023). Applied Environmental
Genomics. CSIRO PUBLISHING.

Bohmann, K., Elbrecht, V., Carøe, C., Bista, I., Leese, F., Bunce, M., ... & Creer, S.
(2022). Strategies for sample labelling and library preparation in DNA metabarcoding
studies. Molecular Ecology Resources, 22(4), 1231-1246.
https://doi.org/10.1111/1755-0998.13512

Bohmann, K., Evans, A., Gilbert, M. T. P., Carvalho, G. R., Creer, S., Knapp, M., ... & De
Bruyn, M. (2014). Environmental DNA for wildlife biology and biodiversity monitoring.
Trends in Ecology & Evolution, 29(6), 358-367.
https://doi.org/10.1016/j.tree.2014.04.003

Bolger, A. M., Lohse, M., & Usadel, B. (2014). Trimmomatic: a flexible trimmer for
Illumina sequence data. Bioinformatics, 30(15), 2114-2120.
https://doi.org/10.1093/bioinformatics/btu170

Bolyen, E., Rideout, J. R., Dillon, M. R., Bokulich, N. A., Abnet, C. C., Al-Ghalith, G. A., ...
& Caporaso, J. G. (2019). Reproducible, interactive, scalable and extensible
microbiome data science using QIIME 2. Nature Biotechnology, 37(8), 852-857.
https://doi.org/10.1038/s41587-019-0209-9

Boyer, F., Mercier, C., Bonin, A., Le Bras, Y., Taberlet, P., & Coissac, E. (2016). obitools:
A unix-inspired software package for DNA metabarcoding. Molecular Ecology
Resources, 16(1), 176-182. https://doi.org/10.1111/1755-0998.12428

Bruce, K., Blackman, R. C., Bourlat, S. J., Hellström, M., Bakker, J., Bista, I., ... & Deiner,
K. (2021). A practical guide to DNA-based methods for biodiversity assessment.
Pensoft Advanced Books. https://doi.org/10.3897/ab.e68634

Brys, R., Halfmaerten, D., Neyrinck, S., Mauvisseau, Q., Auwerx, J., Sweet, M., &
Mergeay, J. (2021). Reliable eDNA detection and quantification of the European
weather loach (Misgurnus fossilis). Journal of Fish Biology, 98(2), 399-414.
https://doi.org/10.1111/jfb.14315

Bustin, S. A., Benes, V., Garson, J. A., Hellemans, J., Huggett, J., Kubista, M., ... &
Wittwer, C. T. (2009). The MIQE guidelines: Minimum information for publication of
quantitative real-time PCR experiments. Clinical Chemistry, 55(4), 611-622.
https://doi.org/10.1373/clinchem.2008.112797

Buxton, A. S., Groombridge, J. J., & Griffiths, R. A. (2018). Seasonal variation in
environmental DNA detection in sediment and water samples. PLoS One, 13(1),
e0191737. https://doi.org/10.1371/journal.pone.0191737

Buxton, A. S., Groombridge, J. J., Zakaria, N. B., & Griffiths, R. A. (2017). Seasonal
variation in environmental DNA in relation to population size and environmental
factors. Scientific Reports, 7(1), 46294. https://doi.org/10.1038/srep46294

Callahan, B. J., McMurdie, P. J., Rosen, M. J., Han, A. W., Johnson, A. J. A., & Holmes, S.
P. (2016). DADA2: High-resolution sample inference from Illumina amplicon data.
Nature Methods, 13(7), 581-583. https://doi.org/10.1038/nmeth.3869

Chen, Y., Tournayre, O., Tian, H., & Lougheed, S. C. (2023). Assessing the breeding
phenology of a threatened frog species using eDNA and automatic acoustic
monitoring. PeerJ, 11, e14679. https://doi.org/10.7717/peerj.14679

Cristescu, M. E. (2019). Can environmental RNA revolutionize biodiversity science?
Trends in Ecology & Evolution, 34(8), 694-697.
https://doi.org/10.1016/j.tree.2019.05.003

Curd, E. E., Gold, Z., Kandlikar, G. S., Gomer, J., Ogden, M., O'Connell, T., ... & Meyer,
R. S. (2019). Anacapa Toolkit: An environmental DNA toolkit for processing multilocus
metabarcode datasets. Methods in Ecology and Evolution, 10(9), 1469-1475.
https://doi.org/10.1111/2041-210X.13214

Davis, C. N., Tyson, F., Cutress, D., Davies, E., Jones, D. L., Brophy, P. M., ... & Jones, R.
A. (2020). Rapid detection of Galba truncatula in water sources on pasture-land using
loop-mediated isothermal amplification for control of trematode infections. Parasites
& Vectors, 13, 1-11. https://doi.org/10.1186/s13071-020-04371-0

Deagle, B. E., Thomas, A. C., McInnes, J. C., Clarke, L. J., Vesterinen, E. J., Clare, E. L., ...
& Eveson, J. P. (2019). Counting with DNA in metabarcoding studies: How should we
convert sequence reads to dietary data?. Molecular Ecology, 28(2), 391-406.
https://doi.org/10.1111/mec.14734

Deiner, K., & Altermatt, F. (2014). Transport distance of invertebrate environmental
DNA in a natural river. PloS One, 9(2), e88786.
https://doi.org/10.1371/journal.pone.0088786

Deiner, K., Bik, H. M., Mächler, E., Seymour, M., Lacoursière-Roussel, A., Altermatt, F.,
... & Bernatchez, L. (2017). Environmental DNA metabarcoding: Transforming how we
survey animal and plant communities. Molecular Ecology, 26(21), 5872-5895.
https://doi.org/10.1111/mec.14350

Deiner, K., Fronhofer, E. A., Mächler, E., Walser, J. C., & Altermatt, F. (2016).
Environmental DNA reveals that rivers are conveyer belts of biodiversity information.
Nature Communications, 7(1), 12544. https://doi.org/10.1038/ncomms12544

Dejean, T., Valentini, A., Duparc, A., Pellier-Cuit, S., Pompanon, F., Taberlet, P., &
Miaud, C. (2011). Persistence of environmental DNA in freshwater ecosystems. PloS
One, 6(8), e23398. https://doi.org/10.1371/journal.pone.0023398

Doi, H., Takahara, T., Minamoto, T., Matsuhashi, S., Uchii, K., & Yamanaka, H. (2015a).
Droplet digital polymerase chain reaction (PCR) outperforms real-time PCR in the
detection of environmental DNA from an invasive fish species. Environmental Science
& Technology, 49(9), 5601-5608. https://doi.org/10.1021/acs.est.5b00253

Doi, H., Uchii, K., Takahara, T., Matsuhashi, S., Yamanaka, H., & Minamoto, T. (2015b).
Use of droplet digital PCR for estimation of fish abundance and biomass in
environmental DNA surveys. PLoS One, 10(3), e0122763.
https://doi.org/10.1371/journal.pone.0122763

Dong, L., Meng, Y., Sui, Z., Wang, J., Wu, L., & Fu, B. (2015). Comparison of four digital
PCR platforms for accurate quantification of DNA copy number of a certified plasmid
DNA reference material. Nature Scientific Reports, 5, 13174.
https://doi.org/10.1038/srep13174

Edgar, R. C. (2010). Search and clustering orders of magnitude faster than BLAST.
Bioinformatics, 26(19), 2460-2461. https://doi.org/10.1093/bioinformatics/btq461

Elbrecht, V., & Leese, F. (2017a). PrimerMiner: an R package for development and in
silico validation of DNA metabarcoding primers. Methods in Ecology and Evolution,
8(5), 622-626. https://doi.org/10.1111/2041-210X.12687

Elbrecht, V., & Leese, F. (2017b). Validation and development of COI metabarcoding
primers for freshwater macroinvertebrate bioassessment. Frontiers in Environmental
Science, 5, 11. https://doi.org/10.3389/fenvs.2017.00011

Elbrecht, V., Taberlet, P., Dejean, T., Valentini, A., Usseglio-Polatera, P., Beisel, J. N., ...
& Leese, F. (2016). Testing the potential of a ribosomal 16S marker for DNA
metabarcoding of insects. PeerJ, 4, e1966. https://doi.org/10.7717/peerj.1966

Escudié, F., Auer, L., Bernard, M., Mariadassou, M., Cauquil, L., Vidal, K., ... & Pascal, G.
(2018). FROGS: find, rapidly, OTUs with galaxy solution. Bioinformatics, 34(8), 1287-
1294. https://doi.org/10.1093/bioinformatics/btx791

Feng, W., Bulté, G., & Lougheed, S. C. (2020). Environmental DNA surveys help to
identify winter hibernacula of a temperate freshwater turtle. Environmental DNA,
2(2), 200-209. https://doi.org/10.1002/edn3.58

Ficetola, G. F., Coissac, E., Zundel, S., Riaz, T., Shehzad, W., Bessière, J., ... &
Pompanon, F. (2010). An in silico approach for the evaluation of DNA barcodes. BMC
Genomics, 11, 1-10. http://www.biomedcentral.com/1471-2164/11/434

Ficetola, G. F., Miaud, C., Pompanon, F., & Taberlet, P. (2008). Species detection using
environmental DNA from water samples. Biology Letters, 4(4), 423-425.
https://doi.org/10.1098/rsbl.2008.0118

Forootan, A., Sjöback, R., Björkman, J., Sjögreen, B., Linz, L., & Kubista, M. (2017).
Methods to determine limit of detection and limit of quantification in quantitative
real-time PCR (qPCR). Biomolecular Detection and Quantification, 12, 1-6.
https://doi.org/10.1016/j.bdq.2017.04.001

Freeland, J. R. (2017). The importance of molecular markers and primer design when
characterizing biodiversity from environmental DNA. Genome, 60(4), 358-374.
https://doi.org/10.1139/gen-2016-0100

Galan, M., Pons, J. B., Tournayre, O., Pierre, E., Leuchtmann, M., Pontier, D., &
Charbonnel, N. (2018). Metabarcoding for the parallel identification of several
hundred predators and their prey: Application to bat species diet analysis. Molecular
Ecology Resources, 18(3), 474-489. https://doi.org/10.1111/1755-0998.12749

Ganguli, A., Mostafa, A., Berger, J., Aydin, M. Y., Sun, F., Ramirez, S. A. S. D., ... &
Bashir, R. (2020). Rapid isothermal amplification and portable detection system for
SARS-CoV-2. Proceedings of the National Academy of Sciences, 117(37), 22727-22735.
https://doi.org/10.1073/pnas.2014739117

Garlapati, D., Charankumar, B., Ramu, K., Madeswaran, P., & Ramana Murthy, M. V.
(2019). A review on the applications and recent advances in environmental DNA
(eDNA) metagenomics. Reviews in Environmental Science and Bio/Technology, 18,
389-411. https://doi.org/10.1007/s11157-019-09501-4

Goldberg, C. S., Turner, C. R., Deiner, K., Klymus, K. E., Thomsen, P. F., Murphy, M. A.,
... & Taberlet, P. (2016). Critical considerations for the application of environmental
DNA methods to detect aquatic species. Methods in Ecology and Evolution, 7(11),
1299-1307. https://doi.org/10.1111/2041-210X.12595

Hakimzadeh, A., Abdala Asbun, A., Albanese, D., Bernard, M., Buchner, D., Callahan,
B., ... & Anslan, S. (2023). A pile of pipelines: An overview of the bioinformatics
software for metabarcoding data analyses. Molecular Ecology Resources.
https://doi.org/10.1111/1755-0998.13847

Hallam, J., Clare, E. L., Jones, J. I., & Day, J. J. (2021). Biodiversity assessment across a
dynamic riverine system: A comparison of eDNA metabarcoding versus traditional fish
surveying methods. Environmental DNA, 3(6), 1247-1266.
https://doi.org/10.1002/edn3.241

Hartman, L. J., Coyne, S. R., & Norwood, D. A. (2005). Development of a novel internal
positive control for Taqman based assays. Molecular and Cellular Probes, 19(1), 51-59.
https://doi.org/10.1016/j.mcp.2004.07.006

Hebert, P. D., Ratnasingham, S., & De Waard, J. R. (2003). Barcoding animal life:
cytochrome c oxidase subunit 1 divergences among closely related species.
Proceedings of the Royal Society of London. Series B: Biological Sciences, 270(suppl_1),
S96-S99. https://doi.org/10.1098/rsbl.2003.0025

Hechler, R. M., & Cristescu, M. E. (2024). Revealing population demographics with
environmental RNA. Molecular Ecology Resources, e13951.
https://doi.org/10.1111/1755-0998.13951

Higuchi, R., Fockler, C., Dollinger, G., & Watson, R. (1993). Kinetic PCR analysis: realtime
monitoring of DNA amplification reactions. Bio/technology, 11(9), 1026-1030.
https://doi.org/10.1038/nbt0993-1026

Hinlo, R., Gleeson, D., Lintermans, M., & Furlan, E. (2017). Methods to maximize
recovery of environmental DNA from water samples. PloS One, 12(6), e0179251.
https://doi.org/10.1371/journal.pone.0179251

Hollingsworth, P. M., Forrest, L. L., Spouge, J. L., Hajibabaei, M., Ratnasingham, S., ... &
Little, D. P. (2009). A DNA barcode for land plants. Proceedings of the National
Academy of Sciences, 106(31), 12794-12797.
https://doi.org/10.1073/pnas.0905845106

Hunter, M. E., Dorazio, R. M., Butterfield, J. S., Meigs-Friend, G., Nico, L. G., &
Ferrante, J. A. (2017). Detection limits of quantitative and digital PCR assays and their
influence in presence–absence surveys of environmental DNA. Molecular Ecology
Resources, 17(2), 221-229. https://doi.org/10.1111/1755-0998.12619

Hunter, M. E., Ferrante, J. A., Meigs-Friend, G., & Ulmer, A. (2019). Improving eDNA
yield and inhibitor reduction through increased water volumes and multi-filter
isolation techniques. Scientific Reports, 9(1), 5259.
https://doi.org/10.1038/s41598-019-40977-w

Janda, J. M., & Abbott, S. L. (2007). 16S rRNA gene sequencing for bacterial
identification in the diagnostic laboratory: pluses, perils, and pitfalls. Journal of Clinical
Microbiology, 45(9), 2761-2764. https://doi.org/10.1128/jcm.01228-07

Jerde, C. L., Mahon, A. R., Chadderton, W. L., & Lodge, D. M. (2011). “Sight-unseen”
detection of rare aquatic species using environmental DNA. Conservation Letters, 4(2),
150-157. https://doi.org/10.1111/j.1755-263X.2010.00158.x

Kagzi, K., Hechler, R. M., Fussmann, G. F., & Cristescu, M. E. (2022). Environmental
RNA degrades more rapidly than environmental DNA across a broad range of pH
conditions. Molecular Ecology Resources, 22(7), 2640-2650.
https://doi.org/10.1111/1755-0998.13655

Kagzi, K., Millette, K. L., Littlefair, J. E., Pochon, X., Wood, S. A., Fussmann, G. F., &
Cristescu, M. E. (2023). Assessing the degradation of environmental DNA and RNA
based on genomic origin in a metabarcoding context. Environmental DNA, 5(5), 1016-
1031. https://doi.org/10.1002/edn3.437

Kamel, B., Laidemitt, M. R., Lu, L., Babbitt, C., Weinbaum, O. L., Mkoji, G. M., & Loker,
E. S. (2021). Detecting and identifying Schistosoma infections in snails and aquatic
habitats: A systematic review. PLoS Neglected Tropical Diseases, 15(3), e0009175.
https://doi.org/10.1371/journal.pntd.0009175

Katz, A. D., Tetzlaff, S. J., Johnson, M. D., Noble, J. D., Rood, S., Maki, D., & Sperry, J. H.
(2023). Molecular identification and environmental DNA detection of gill lice
ectoparasites associated with Brook Trout declines. Transactions of the American
Fisheries Society, 152(6), 788-808. https://doi.org/10.1002/tafs.10439

Keck, F., Blackman, R. C., Bossart, R., Brantschen, J., Couton, M., Hürlemann, S.,
Kirschner, D., Locher, N., Zhang, H., & Altermatt, F. (2021). Meta-analysis shows both
congruence and complementarity of DNA metabarcoding to traditional methods for
biological community assessment. bioRxiv, 2021-06.
https://doi.org/10.1101/2021.06.29.450286

Kelly, R. P., Shelton, A. O., & Gallego, R. (2019). Understanding PCR processes to draw
meaningful conclusions from environmental DNA studies. Scientific Reports, 9(1),
12133. https://doi.org/10.1038/s41598-019-48546-x

Kirse, A., Bourlat, S. J., Langen, K., & Fonseca, V. G. (2021). Metabarcoding malaise
traps and soil eDNA reveals seasonal and local arthropod diversity shifts. Scientific
Reports, 11(1), 10498. https://doi.org/10.1038/s41598-021-89950-6

Klymus, K. E., Merkes, C. M., Allison, M. J., Goldberg, C. S., Helbing, C. C., Hunter, M.
E., Jackson, C. A., Lance, R. F., Mangan, A. M., Monroe, E. M., Piaggio, A. J., Stokdyk, J.
P., Wilson, C.C., & Richter, C. A. (2020a). Reporting the limits of detection and
quantification for environmental DNA assays. Environmental DNA, 2(3), 271-282.
https://doi.org/10.1002/edn3.29

Klymus, K. E., Ramos, D. V. R., Thompson, N. L., & Richter, C. A. (2020b). Development
and testing of species-specific quantitative PCR assays for environmental DNA
applications. JoVE (Journal of Visualized Experiments), (165), e61825.
https://doi.org/10.3791/61825

Kozich, J. J., Westcott, S. L., Baxter, N. T., Highlander, S. K., & Schloss, P. D. (2013).
Development of a dual-index sequencing strategy and curation pipeline for analyzing
amplicon sequence data on the MiSeq Illumina sequencing platform. Applied and
Environmental Microbiology, 79(17), 5112-5120. https://doi.org/10.1128/AEM.01043-

Kronenberger, J. A., Wilcox, T. M., Mason, D. H., Franklin, T. W., McKelvey, K. S.,
Young, M. K., & Schwartz, M. K. (2022). eDNAssay: A machine learning tool that
accurately predicts qPCR cross-amplification. Molecular Ecology Resources, 22(8),
2994-3005. https://doi.org/10.1111/1755-0998.13681

Kubista, M. Prime time for qPCR – Raising the quality bar. (2014). European
Pharmaceutical Review, 19(3), 63–67.

Lamb, P. D., Hunter, E., Pinnegar, J. K., Creer, S., Davies, R. G., & Taylor, M. I. (2019).
How quantitative is metabarcoding: A meta-analytical approach. Molecular Ecology,
28(2), 420-430. https://doi.org/10.1111/mec.14920

Langlois, V. S., Allison, M. J., Bergman, L. C., To, T. A., & Helbing, C. C. (2021). The need
for robust qPCR-based eDNA detection assays in environmental monitoring and
species inventories. Environmental DNA, 3(3), 519-527.
https://doi.org/10.1002/edn3.164

Lawson Handley, L., Read, D. S., Winfield, I. J., Kimbell, H., Johnson, H., Li, J., Hanh, C.,
Blackman, R., Wilcox, R., Donnelly, R., Szitenberg, A., & Hänfling, B. (2019). Temporal
and spatial variation in distribution of fish environmental DNA in England’s largest
lake. Environmental DNA, 1(1), 26-39. https://doi.org/10.1002/edn3.5

Littlefair, J. E., Rennie, M. D., & Cristescu, M. E. (2022). Environmental nucleic acids: A
field-based comparison for monitoring freshwater habitats using eDNA and eRNA.
Molecular Ecology Resources, 22(8), 2928-2940.
https://doi.org/10.1111/1755-0998.13671

Macher, T. H., Arle, J., Beermann, A. J., Frank, L., Hupało, K., Koschorreck, J., ... &
Leese, F. Is it worth the extra mile? Comparing environmental DNA and RNA
metabarcoding for vertebrate and invertebrate biodiversity surveys in a lowland
stream. DNA metabarcoding for the ecological status assessment in streamsvalidation,
plausibility check and intercalibration of the new assessment method, PhD
thesis, Universität Duisburg-Essen, Chapter 4.3, 135-154.

Magoč, T., & Salzberg, S. L. (2011). FLASH: fast length adjustment of short reads to
improve genome assemblies. Bioinformatics, 27(21), 2957-2963.
https://doi.org/10.1093/bioinformatics/btr507

Mahé, F., Rognes, T., Quince, C., De Vargas, C., & Dunthorn, M. (2015). Swarm v2:
highly-scalable and high-resolution amplicon clustering. PeerJ, 3, e1420.
https://doi.org/10.7717/peerj.1420

Mahon, A. R., Jerde, C. L., Galaska, M., Bergner, J. L., Chadderton, W. L., Lodge, D. M.,
Hunter, M. E., & Nico, L. G. (2013). Validation of eDNA surveillance sensitivity for
detection of Asian carps in controlled and field experiments. PLoS One, 8(3), e58316.
https://doi.org/10.1371/journal.pone.0058316

Majaneva, M., Diserud, O. H., Eagle, S. H., Boström, E., Hajibabaei, M., & Ekrem, T.
(2018). Environmental DNA filtration techniques affect recovered biodiversity.
Scientific Reports, 8(1), 4682. https://doi.org/10.1038/s41598-018-23052-8

Mao, X., Liu, C., Tong, H., Chen, Y., & Liu, K. (2019). Principles of digital PCR and its
applications in current obstetrical and gynecological diseases. American Journal of
Translational Research, 11(12), 7209.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6943456/

Maracle, S. R., Tournayre, O., Windle, M. J., Cormier, E., Schwartz, K., Wylie-Arbic, M.,
Rundle, E., Perron, M. A., Francis, A., & Lougheed, S. C. (2024). Nearshore fish diversity
changes with sampling method and human disturbance: Comparing eDNA
metabarcoding and seine netting along the Upper St. Lawrence River. Journal of Great
Lakes Research, 102317. https://doi.org/10.1016/j.jglr.2024.102317

Marshall, N. T., Vanderploeg, H. A., & Chaganti, S. R. (2021). Environmental (e)RNA
advances the reliability of eDNA by predicting its age. Scientific Reports, 11(1), 2769.
https://doi.org/10.1038/s41598-021-82205-4

Martin, D. I., Ross, R., Quetin, L., & Murray, A. (2006). Molecular approach (PCRDGGE)
to diet analysis in young Antarctic krill Euphausia Superba. Marine Ecology
Progress Series, 319 (August), 155–65. https://doi.org/10.3354/meps319155

Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput
sequencing reads. EMBnet.Journal, 17(1), 10–12.
https://doi.org/10.14806/ej.17.1.200

Mathon, L., Valentini, A., Guérin, P., Normandeau, E., Noel, C., Lionnet, C., Boulanger,
E., Thuiller, W., Bernatchez, L., Mouillot, D., Dejean, T., & Manel, S. (2021).
Benchmarking bioinformatic tools for fast and accurate eDNA metabarcoding species
identification. Molecular Ecology Resources, 21(7), 2565–79.
https://doi.org/10.1111/1755-0998.13430

Mauvisseau, Q., Davy-Bowker, J., Bulling, M., Brys, R., Neyrinck, S., Troth, C., & Sweet,
M. (2019). Combining ddPCR and environmental DNA to improve detection
capabilities of a critically endangered freshwater invertebrate. Scientific Reports, 9(1),
14064. https://doi.org/10.1038/s41598-019-50571-9

Mauvisseau, Q., Harper, L. R., Sander, M., Hanner, R.H., Kleyer, H., & Deiner, K. (2022).
The multiple states of environmental DNA and what is known about their persistence
in aquatic environments. Environmental Science & Technology, 56(9): 5322-5333.
https://doi.org/10.1021/acs.est.1c07638

McKee, A. M., Spear, S. F., & Pierson, T. D. (2015). The effect of dilution and the use of
a post-extraction nucleic acid purification column on the accuracy, precision, and
inhibition of environmental DNA samples. Biological Conservation, 183, 70-76.
https://doi.org/10.1016/j.biocon.2014.11.031

Mendoza, M. L. Z., Sicheritz-Ponten, T., & Gilbert, M. T. P. (2015). Environmental
genes and genomes: understanding the differences and challenges in the approaches
and software for their analyses. Briefings in Bioinformatics, 16(5), 745-758.
https://doi.org/10.1093/bib/bbv001

Milla, L., Schmidt-Lebuhn, A., Bovill, J., & Encinas-Viso, F. (2022). Monitoring of honey
bee floral resources with pollen DNA metabarcoding as a complementary tool to
vegetation surveys. Ecological Solutions and Evidence, 3(1), e12120.
https://doi.org/10.1002/2688-8319.12120

Milligan, J. N., Shroff, R., Garry, D. J., & Ellington, A. D. (2018). Evolution of a
thermophilic strand-displacing polymerase using high-temperature isothermal
compartmentalized self-replication. Biochemistry, 57(31), 4607–19.
https://doi.org/10.1021/acs.biochem.8b00200

Mishra, S., Goyal, D., & Phurailatpam, L. (2021). Targeted 16S rRNA gene and ITS2
amplicon sequencing of leaf and spike tissues of Piper longum identifies new
candidates for bioprospecting of bioactive compounds. Archives of Microbiology,
203(7), 3851-3867. https://doi.org/10.1007/s00203-021-02356-w

Miya, M., Sato, Y., Fukunaga, T., Sado, T., Poulsen, J. Y., Sato, K., Minamoto, T.,
Yamanaka, H., Araki, H., & Iwasaki, W. (2015). MiFish, a set of universal PCR primers
for metabarcoding environmental DNA from fishes: Detection of more than 230
subtropical marine species. Royal Society Open Science, 2(7), 150088.
https://doi.org/10.1098/rsos.150088

Mlaga, K. D., Mathieu, A., Beauparlant, C. J., Ott, A., Khodr, A., Perin, O., & Droit, A.
(2021). HCK and ABAA: A newly designed pipeline to improve fungi metabarcoding
analysis. Frontiers in Microbiology, 12, 640693.
https://doi.org/10.3389/fmicb.2021.640693

Mori, Y., Nagamine, K., Tomita, N., & Notomi, T. (2001). Detection of loop-mediated
isothermal amplification reaction by turbidity derived from magnesium
pyrophosphate formation. Biochemical and Biophysical Research Communications,
289(1), 150-154. https://doi.org/10.1006/bbrc.2001.5921

Nathan, L. M., Simmons, M., Wegleitner, B. J., Jerde, C. L., and Mahon, A. R. (2014).
Quantifying environmental DNA signals for aquatic invasive species across multiple
detection platforms. Environmental Science & Technology, 48(21), 12800–806.
https://doi.org/10.1021/es5034052

Nordstrom, B, Budd, A., Mitchell, N., Cornish, C., Byrne, M., Kuchling, G., & Jarman, S.
(2023). Environmental DNA reflects spatial distribution of a rare turtle in a lentic
wetland assisted colonization site. Environmental DNA, 6(1), e507.
https://doi.org/10.1002/edn3.507

Nørgaard, L., Olesen, C. R., Trøjelsgaard, K., Pertoldi, C., Nielsen, J. L., Taberlet, P.,
Ruiz-González, A., De Barba, M., & Iacolina, L. (2021). eDNA metabarcoding for
biodiversity assessment, generalist predators as sampling assistants. Scientific
Reports, 11(1), 6820. https://doi.org/10.1038/s41598-021-85488-9

Notomi, T., Okayama, H., Masubuchi, H., Yonekawa, T., Watanabe, K., Amino, N., &
Hase, T. (2000). Loop-mediated isothermal amplification of DNA. Nucleic Acids
Research, 28(12), e63. https://doi.org/10.1093/nar/28.12.e63

Oehm, J., Juen, A., Nagiller, K., Neuhauser, S., & Traugott, M. (2011). Molecular
scatology: How to improve prey DNA detection success in avian faeces? Molecular
Ecology Resources, 11(4), 620–628.
https://doi.org/10.1111/j.1755-0998.2011.03001.x

Ogram, A., Sayler, G. S., & Barkay, T. (1987). The extraction and purification of
microbial DNA from sediments. Journal of Microbiological Methods, 7(2), 57–66.
https://doi.org/10.1016/0167-7012(87)90025-X

Olson, Z. H., Briggler, J. T., Williams, R. N., Olson, Z. H., Briggler, J. T., & Williams, R. N.
(2012). An eDNA approach to detect Eastern Hellbenders (Cryptobranchus a.
Alleganiensis) using samples of water. Wildlife Research, 39(7), 629–636.
https://doi.org/10.1071/WR12114

Osathanunkul, M., & Suwannapoom, C. (2024). A comparative study on eDNA-based
detection of Siamese bat catfish (Oreoglanis siamensis) in wet and dry conditions.
Scientific Reports, 14(1), 8885. https://doi.org/10.1038/s41598-024-58752-x

Parsley, M. B., & Goldberg, C. S. (2023). Environmental RNA can distinguish life stages
in amphibian populations. Molecular Ecology Resources, 24(4), e13857.
https://doi.org/10.1111/1755-0998.13857

Pawlowski, J., Apothéloz-Perret-Gentil, L., & Altermatt, F. (2020). Environmental DNA:
What’s behind the term? Clarifying the terminology and recommendations for its
future use in biomonitoring. Molecular Ecology, 29(22), 4258–4264.
https://doi.org/10.1111/mec.15643

Porter, T. M., and Hajibabaei, M. (2018). Scaling up: A guide to high-throughput
genomic approaches for biodiversity analysis. Molecular Ecology, 27(2), 313–338.
https://doi.org/10.1111/mec.14478

Puechmaille, S. J., & Teeling, E. C. (2014). Non-invasive genetics can help find rare
species: A case study with Rhinolophus Mehelyi and R. Euryale (Rhinolophidae:
Chiroptera) in Western Europe. Mammalia, 78(2), 251–255.
https://doi.org/10.1515/mammalia-2013-0040

Rådström, P., Knutsson, R., Wolffs, P., Lövenklev, M., & Löfström, C. (2004). Pre-PCR
processing: strategies to generate PCR-compatible samples. Molecular Biotechnology,
26: 133-146. https://doi.org/10.1385/mb:26:2:133

Ratnasingham, S., & Hebert, P. D. N. (2007). BOLD: The barcode of life data system
(http://www.barcodinglife.org). Molecular Ecology Notes, 7(3), 355–364.
https://doi.org/10.1111/j.1471-8286.2007.01678.x

Ririe, K. M., Rasmussen, R. P., Wittwer, C. T. (1997). Product differentiation by analysis
of DNA melting curves during the polymerase chain reaction. Analytical Biochemistry,
245(2), 154-160. https://doi.org/10.1006/abio.1996.9916

Rognes, T., Flouri, T., Nichols, B., Quince, C., & Mahé, F. (2016). VSEARCH: A versatile
open source tool for metagenomics. PeerJ, 4(October), e2584.
https://doi.org/10.7717/peerj.2584

Rourke, M. L., Fowler, A. M., Hughes, J. M., Broadhurst, M. K., DiBattista, J. D., Fielder,
S., Walburn, J. W., & Furlan, E. M. (2022). Environmental DNA (eDNA) as a tool for
assessing fish biomass: A review of approaches and future considerations for resource
surveys. Environmental DNA, 4(1), 9–33. https://doi.org/10.1002/edn3.185

Ruppert, K. M., Kline, R. J., & Rahman, M. S. (2019). Past, present, and future
perspectives of environmental DNA (eDNA) metabarcoding: A systematic review in
methods, monitoring, and applications of global eDNA. Global Ecology and
Conservation, 17(January), e00547. https://doi.org/10.1016/j.gecco.2019.e00547

Sahoo, P. R., Sethy, K., Mohapatra, S., & Panda, D. (2016). Loop mediated isothermal
amplification: An innovative gene amplification technique for animal diseases.
Veterinary World, 9(5), 465–469. https://doi.org/10.14202/vetworld.2016.465-469

Schenk, J., Geisen, S., Kleinbölting, N., & Traunspurger, W. (2019). Metabarcoding data
allow for reliable biomass estimates in the most abundant animals on Earth.
Metabarcoding and Metagenomics, 3, e46704.
https://doi.org/10.3897/mbmg.3.46704

Schloss, P. D., Gevers, D., & Westcott, S. L. (2011). Reducing the effects of PCR
amplification and sequencing artifacts on 16S rRNA-based studies. PLoS ONE, 6(12),
e27310. https://doi.org/10.1371/journal.pone.0027310.

Seki, M., Kilgore, P. E., Kim, E. J., Ohnishi, M., Hayakawa, S., Kim, D. W. (2018). Loopmediated
isothermal amplification methods for diagnosis of bacterial meningitis.
Frontiers in Pediatrics, 6, 2296-2360. https://doi.org/10.3389/fped.2018.00057

Sengupta, M. E., Hellström, M., Kariuki, H. C., Olsen, A., Thomsen, P. F., Mejer, H.,
Willerslev, E., Mwanje, M. T., Madsen, H., Kristensen, T. K., Stensgaard, A., &
Vennervald, B. J. (2019). Environmental DNA for improved detection and
environmental surveillance of schistosomiasis. Proceedings of the National Academy
of Sciences, 116(18), 8931–8940. https://doi.org/10.1073/pnas.1815046116

Shogren, A. J., Tank, J. L., Andruszkiewicz, E., Olds, B., Mahon, A. R., Jerde, C. L., &
Bolster, D. 2017. Controls on eDNA movement in streams: Transport, retention, and
resuspension. Scientific Reports, 7(1), 5065.
https://doi.org/10.1038/s41598-017-05223-1

Singer, G. A., Shekarriz, S., McCarthy, A., Fahner, N., & Hajibabaei, M. (2020). The
utility of a metagenomics approach for marine biomonitoring. BioRxiv, 2020-03.
https://doi.org/10.1101/2020.03.16.993667

Slatko, B. E., Gardner, A. F., & Ausubel, F. M. (2018). Overview of next-generation
sequencing technologies. Current Protocols In Molecular Biology, 122(1), e59.
https://doi.org/10.1002/cpmb.59

Soraka, M., Wasowicz, B., & Rymaszewska, A. (2021). Loop-mediated isothermal
amplification (LAMP): The better sibling of PCR. Cells, 10(8), 1931.
https://doi.org/10.3390%2Fcells10081931

Stewart, K. A. (2019). Understanding the effects of biotic and abiotic factors on
sources of aquatic environmental DNA. Biodiversity and Conservation, 28(5), 983–
1001. https://doi.org/10.1007/s10531-019-01709-8

Strickler, K. M., Fremier, A. K., & Goldberg, C. S. (2015). Quantifying effects of UV-B,
temperature, and pH on eDNA degradation in aquatic microcosms. Biological
Conservation, Special Issue: Environmental DNA: A powerful new tool for biological
conservation, 183(March), 85–92. https://doi.org/10.1016/j.biocon.2014.11.038

Taberlet, P, Bonin, A., Zinger, L., & Coissac, E. (2018). Environmental DNA: For
Biodiversity Research and Monitoring. Illustrated edition. Oxford, United Kingdom:
Oxford University Press.

Taberlet, P, Coissac, E., Hajibabaei, & Rieseberg, M. L. H. (2012). Environmental DNA.
Molecular Ecology, 21(8), 1789–1793. https://doi.org/10.1111/j.1365-294X.2012.05542.x

Takasaki, K., Aihara, H., Imanaka, T., Matsudaira, T., Tsukahara, K., Usui, A., Osaki, S.,
& Doi, H. (2021). Water pre-filtration methods to improve environmental DNA
detection by real-time PCR and metabarcoding. PLoS ONE, 16(5), e0250162.
https://doi.org/10.1371/journal.pone.0250162

Tanner, N. A., Zhang, Y., & Evans Jr., T. C. (2015). Visual detection of isothermal nucleic
acid amplification using pH-sensitive dyes. Biotechniques, 58(2), 59-68.
https://doi.org/10.2144/000114253

Thalinger, B., Rieder, A., Teuffenbach, A., Pütz, Y., Schwerte, T., Wanzenböck, J., &
Traugott, M. (2021a). The effect of activity, energy use, and species identity on
environmental DNA shedding of freshwater fish. Frontiers in Ecology and Evolution, 9,
623718. https://doi.org/10.3389/fevo.2021.623718

Thalinger, B., Deiner, K., Harper, L. R., Rees, H. C., Blackman, R. C., Sint, D., Traugott,
M., Goldberg, C. S., & Bruce, K. (2021b). A validation scale to determine the readiness
of environmental DNA assays for routine species monitoring. Environmental DNA,
3(4), 823–836. https://doi.org/10.1002/edn3.189

Thomsen, P. F., & Willerslev, E. (2015). Environmental DNA – An emerging tool in
conservation for monitoring past and present biodiversity. Biological Conservation,
Special Issue: Environmental DNA: A Powerful New Tool for Biological Conservation,
183 (Supplement C), 4–18. https://doi.org/10.1016/j.biocon.2014.11.019

Thomsen, P. F., Kielgast, J., Iversen, L. L., Møller, P. R., Rasmussen, M., & Willerslev, E.
(2012). Detection of a diverse marine fish fauna using environmental DNA from
seawater samples. PLoS ONE, 7(8), e41732.
https://doi.org/10.1371/journal.pone.0041732

Thongjued, K., Chotigeat, W., Bumrungsri, S., Thanakiatkrai, P., & Kitpipit, T. (2021).
Direct PCR-DGGE technique reveals Wrinkle-Lipped Free-Tailed Bat (Chaerephon
Plicatus Buchanan, 1800) predominantly consume planthoppers and mosquitoes in
central Thailand. Acta Chiropterologica, 23(1), 93–106.
https://doi.org/10.3161/15081109ACC2021.23.1.008

Tournayre, O., Tian, H., Lougheed, D. R., Windle, M. J. S., Lambert, S., Carter, J., Sun,
Z., Ridal, J., Wang, Y., Cumming, B. F., Arnott, S. E., & Lougheed, S. C. 2023b. How to
barcode (almost all) freshwater biodiversity. bioRxiv: 2023-12.
https://doi.org/10.1101/2023.12.13.571596

Tournayre, O., Leuchtmann, M., Filippi-Codaccioni, O., Trillat, M., Piry, S., Pontier, D.,
Charbonnel, N., & Galan, M. (2020). In silico and empirical evaluation of twelve
metabarcoding primer sets for insectivorous diet analyses. Ecology and Evolution,
10(13), 6310–6332. https://doi.org/10.1002/ece3.6362

Tournayre, O., Wolfe, R., McCurdy-Adams, H., Chabot, A. A., & Lougheed, S. C.
(2023a). A species-specific digital PCR assay for the endangered blue racer (Coluber
constrictor foxii) in Canada. Genome, 66(9). 251-260. https://doi.org/10.1139/gen-
2023-0008

Turner, C. R., Barnes, M. A., Xu, C. C. Y., Jones, S. E., Jerde, C. L., & Lodge, D. M. (2014).
Particle size distribution and optimal capture of aqueous macrobial eDNA. Methods in
Ecology and Evolution, 5(7), 676–684. https://doi.org/10.1111/2041-210X.12206

Vanderpool, D. D., Wilcox T. M., Young, M. K., Pilgrim, K. L., & Schwartz, M. K. (2024).
Simultaneous species detection and discovery with environmental DNA
metabarcoding: A freshwater mollusk case study. Ecology and Evolution, 14(2),
e11020. https://doi.org/10.1002/ece3.11020

Vythalingam, L. M., Hossain, M. A. M., & Bhassu, S. (2021). Rapid in-situ detection kit
(RisK): Development of loop-mediated isothermal amplification (LAMP) assay for the
rapid identification of selected invasive alien fish in Malaysian freshwaters. Molecular
and Cellular Probes, 55 (February), 101683.
https://doi.org/10.1016/j.mcp.2020.101683

Walz, K., Yamahara, K., Michisaki, R., & Chavez, F. 2019. Environmental DNA (eDNA)
extraction using Qiagen DNeasy Blood and Tissue Kit V.2. Protocols.io.
https://doi.org/10.17504/protocols.io.n2udgew

Wilcox, T. M., McKelvey, K. S., Young, M. K., Jane, S. F., Lowe, W. H., Whiteley, A. R., &
Schwartz, M. K. (2013). Robust detection of rare species using environmental DNA:
The importance of primer specificity. PLoS ONE, 8(3), e59520.
https://doi.org/10.1371/journal.pone.0059520

Willerslev, E., Hansen, A. J., Binladen, J., Brand, T. B., Gilbert, M. T. P., Shapiro, B.,
Bunce, M., Wiuf, C., Gilichinsky, D. A., & Cooper, A. (2003). Diverse plant and animal
genetic records from Holocene and Pleistocene sediments. Science, 300(5620), 791–
795. https://doi.org/10.1126/science.1084114

Williams, M. R., Stedtfeld, R. D., Engle, C., Salach, P., Fakher, U., Stedtfeld, T., Dreelin,
E., Stevenson, R. J., Latimore, J., & Hashsham, S. A. (2017). Isothermal amplification of
environmental DNA (eDNA) for direct field-based monitoring and laboratory
confirmation of Dreissena sp. PLoS ONE, 12(10), e0186462.
https://doi.org/10.1371/journal.pone.0186462

Xia, Z., Johansson, M. L., Gao, Y., Zhang, L., Haffner, G. D., MacIsaac, H. J., & Zhan, A.
(2018). Conventional versus real-time quantitative PCR for rare species detection.
Ecology and Evolution, 8(23), 11799–11807. https://doi.org/10.1002/ece3.4636

Yates, M. C., Derry, A. M. & Cristescu, M. E. (2021). Environmental RNA: a revolution
in ecological resolution? Trends in Ecology & Evolution, 36(7), 601-609.
https://doi.org/10.1016/j.tree.2021.03.001

Yu, J., Young, R. G., Deeth, L. E., & Hanner, R. H. (2020). Molecular detection mapping
and analysis platform for R (MDMAPR) facilitating the standardization, analysis,
visualization, and sharing of qPCR data and metadata. PeerJ, 8, e9974.
https://doi.org/10.7717/peerj.9974.

Zhang, C., Yan, L., Wang, L., Jin, Y. J., Chen, L., Shi, Y., & Wang, Q. 2015. The
development and application of digital PCR (in Chinese). Fudan University Journal of
Medical Science, 42, 786–789.

Zhang, D-X., & Hewitt, G. M. (1997). Assessment of the universality and utility of a set
of conserved mitochondrial COI primers in insects. Insect Molecular Biology, 6(2), 143-
150. https://doi.org/10.1111/j.1365-2583.1997.tb00082.x

Zhang, J., Kobert, K., Flouri, T., & Stamatakis, A. (2014). PEAR: A fast and accurate
Illumina paired-end read merger. Bioinformatics, 30(5): 614–620.
https://doi.org/10.1093/bioinformatics/btt593

Zhu, G., Ye, X., Dong, Z., Lu, Y. C., Sun, Y., Liu, Y., McCormack, R., Gu, Y., & Liu, X.
(2015). Highly sensitive droplet digital PCR method for detection of EGFR-activating
mutations in plasma cell-free DNA from patients with advanced non-small cell lung
cancer. Journal of Molecular Diagnostics, 17, 265–272.
https://doi.org/10.1016/j.jmoldx.2015.01.004

Zhu, T., & Iwasaki, W. (2023). MultiBarcodeTools: Easy selection of optimal primers for
eDNA multi-metabarcoding. Environmental DNA, 5(6), 1793-1808.
https://doi.org/10.1002/edn3.499.499