=============================================
Refining the snapshot using environmental RNA
=============================================

One concern with eDNA approaches is that the eDNA signal may persist in the
environment for a long period of time, and can derive from decaying dead organisms or
organisms that are no longer present in the environment (Marshall et al. 2021). This leads to
false positive detections (at least in the temporal sense as the organism was indeed present
at one time) and may constrain assessments of rapid changes in communities.

Environmental RNA (eRNA) has received increasing attention over the past few years
to assess local and living communities. Indeed, eRNA reflects metabolically active assemblages
and thus has potentially a higher turnover than eDNA (Cristescu 2019; Yates et al. 2021). eRNA
is less stable than eDNA and thus degrades faster (Kagzi et al. 2022), but can still be filtered
from water and metabarcoded to reliably provide species composition information (Littlefair
et al. 2022). Its potential goes beyond species detections (see Hechler and Cristescu, 2024).
RNA plays a crucial role in gene expression, so it offers the possibility to assess the health of
ecosystems by analysing the transcriptional profiles of environmental samples
(metatranscriptomics) (Yates et al. 2021), and can even be used to distinguish life stages when
target genes are exclusively expressed in one life stage (Parsley and Goldberg 2023).

The genomic origin (mitochondrial vs nuclear) and marker characteristics (e.g.
ribosomal vs protein-coding gene) are important factors to consider in eRNA work (Marshall
et al. 2021; Kagzi et al. 2023). Mitochondrial genome copies (e.g. COI) are more numerous
than nuclear copies (e.g. except for repetitive nuclear ribosomal markers such as 18S), but
mRNA is single-stranded so less stable than rRNA (transcribed rRNA make stable ribosome
complex with ribosomal proteins). Finally, eRNA studies are still limited by higher cost and
labour than eDNA based studies because eRNA must be processed with extra-care and an
expensive additional step (reverse transcription from RNA to cDNA) (Macher 2023).